akka-http-testkit/src/main/scala/akka/http/scaladsl/testkit/RouteTestResultComponent.scala

```diff
@@ -47,6 +47,12 @@ trait RouteTestResultComponent {
         case _                             => Nil
       }
 
+    def chunksStream: Source[ChunkStreamPart, Any] =
+      rawResponse.entity match {
+        case HttpEntity.Chunked(_, data) => data
+        case _                           => Source.empty
```

Comment:
hmm, perhaps this should throw an exception?

Reply:
I can do that. What exception do you want this to throw?
I returned an empty source for consistency since the method `chunks` returns an empty List in the case the entity is not `Chunked`

Reply:
> I returned an empty source for consistency since the method `chunks` returns an empty `List` in the case the entity is not Chunked

I see, you're right, that makes sense.

You still won't be able to inspect any other aspects of the response except for the chunkStream, though, since those still all go through `entity` and thus `awaitAllElements`, right? How do you think we should support that? We could probably adapt the methods that don't touch the entity's data to look at `rawResponse` instead, or provide access to `rawResponse` somehow?

Reply:
You are right, there are several methods that do not require the response data to be consumed in order for them to be available. 

I've made the necessary changes.